tests/Jackalope/PropertyTest.php

```diff
@@ -2,8 +2,67 @@
 
 namespace Jackalope;
 
+use PHPCR\PropertyType;
+
 class PropertyTest extends TestCase
 {
+    public function provideGetNode()
+    {
+        return array(
+            array(
+                array('1234-1234', '4321-4321'),
+                array(),
+                'Internal Error: Could not find a referenced node. If the referencing node is a frozen version, this can happen, otherwise it would be a bug.',
+            ),
+            array(
+                array('1234-1234', '4321-4321'),
+                array('1234-1234'),
+                'Internal Error: Could not find a referenced node. If the referencing node is a frozen version, this can happen, otherwise it would be a bug.',
+            ),
+            array(
+                array('1234-1234', '4321-4321'),
+                array('1234-1234', '4321-4321'),
+            ),
+            array(
+                array('1234-1234', '4321-4321'),
+                array('4321-4321', '1234-1234'),
+                'Internal Error: Could not find a referenced node. If the referencing node is a frozen version, this can happen, otherwise it would be a bug.',
```

Comment:
Found the reason, it is that on master the check at https://github.com/jackalope/jackalope/blob/1.1/src/Jackalope/Property.php#L450 has been removed. Actually that's exactly what makes our application fail, but we really need the correct order.